Aprendizado por Reforço Clássico/Temporal-Difference/README.md

```diff
@@ -1,5 +1,7 @@
 # 📅 Temporal-Difference Learning
 
+Métodos de **Temporal-Difference** são algoritmos de Aprendizado por Reforço que aprendem utilizando principalmente técnicas de *bootstrapping* (uma estimativa a base de amostragem) da sua função de valor. Esses métodos amostram do ambiente (assim como métodos de Monte Carlo), mas fazem atualizações com base em estimativas atuais. Ou seja, enquanto métodos de Monte Carlo atualizam apenas quando a saída final já é sabida, **Temporal-Difference** atualiza as predições para representarem valores mais tardios sobre o futuro antes do resultado final já ser conhecido.
```

Comment:
Acho que o texto fica melhor assim. Talvez dê pra melhorar um pouco mais, mas não sei como.

Obs: se for aceitar a sugestão, lembrar de copiar pro README de RL clássico.

```suggestion
Métodos de **Temporal-Difference** são algoritmos de Aprendizado por Reforço que aprendem utilizando principalmente técnicas de *bootstrapping* (uma estimativa a base de amostragem) da sua função de valor. Esses métodos amostram do ambiente, assim como métodos de Monte Carlo, mas fazem atualizações com base em estimativas intermediárias. Ou seja, enquanto métodos de Monte Carlo atualizam as suas predições apenas no fim do episódio (quando todos os retornos já são conhecidos), métodos de  **Temporal-Difference** atualizam as suas predições a cada instante de tempo, utilizando *bootstrapping* para estimar o retorno do episódio.
```